src/arr.ts

```diff
@@ -124,6 +124,7 @@ async function makeArrApiCall<ResponseType>(
 	uArrL: string,
 	resourcePath: string,
 	params = new URLSearchParams(),
+	arrInstance?: string,
 ): Promise<Result<ResponseType, Error>> {
 	const apikey = getApikey(uArrL)!;
```

Comment:
```suggestion
	options: { params?: URLSearchParams, arrInstance?: string }
): Promise<Result<ResponseType, Error>> {
	const urlParams = params ?? new URLSearchParams();
	const apikey = getApikey(uArrL)!;
```

Reply:
I think this makes calling it easier to understand

Reply:
check out what I did....

Reply:
yea, saw you edited it :P